### ISSUE TITLE
Move TLDR docs lookup to native

### ISSUE THREAD

**Issue by martonlanga** [TermyApp/termy#87]
#### Move this part:
https://github.com/TermyApp/termy/blob/a6ebb77688230043775a75383ac1f57f3b8e2768/app/src/utils/monaco.ts#L36-L44

#### To here:
https://github.com/TermyApp/termy/blob/e66408851eb5a0d9dd7f7fe213d18a949dcbd64f/native/src/suggestions/executables.rs#L36

#### This can be deleted (also the associated 'tldr' types in `app/types/shared`):
https://github.com/TermyApp/termy/blob/e66408851eb5a0d9dd7f7fe213d18a949dcbd64f/electron/src/ipc.ts#L46-L64

TLDR docs are in the external dir, as a git submodule. Note that the dev and production will be different (you can check current env using `if cfg!(debug_assertions)`)

@FerretCode
